packages/combo-box/theme/lumo/vaadin-combo-box-overlay-styles.js

```diff
@@ -1,7 +1,6 @@
 import '@vaadin/vaadin-lumo-styles/color.js';
 import '@vaadin/vaadin-lumo-styles/spacing.js';
 import '@vaadin/vaadin-lumo-styles/style.js';
-import '@vaadin/overlay/theme/lumo/vaadin-overlay.js';
```

Comment:
This theme import was added to overlay based components in #3888 but it's no longer needed.
The reason is that `vaadin-overlay` does not get defined when importing `vaadin-combo-box`.